src/services/api/search.js

```diff
@@ -2,12 +2,18 @@ import { apiBase } from 'services/config'
 import { SEARCH, SITES } from 'common/constants'
 
 const search = {
-  get: async ({ sitename, searchParams, pageParam, perPage = 48 }) =>
-    apiBase()
-      .get(
-        `${SITES}/${sitename}/${SEARCH}/?${searchParams}&page=${pageParam}&pageSize=${perPage}`,
-      )
-      .json(),
+  get: async ({ sitename, searchParams, pageParam, perPage = 48 }) => {
+    if (sitename) {
+      return apiBase()
+        .get(
+          `${SITES}/${sitename}/${SEARCH}/?${searchParams}&page=${pageParam}&pageSize=${perPage}`,
+        )
+        .json()
+    }
+    return apiBase()
+      .get(`${SEARCH}/?${searchParams}&page=${pageParam}&pageSize=${perPage}`)
+      .json()
+  },
```

Comment:
These api functions map 1:1 with the URL you are calling, and I think it would be beneficial to keep that up.

If we want a magic search function that guesses the URL you want based on the sitename, I think it should be an additional function rather than a replacement for the two simple get() functions.